lib/auto_evaluation/answer_relevancy/reason_generator.rb

```diff
@@ -0,0 +1,44 @@
+module AutoEvaluation
+  class AnswerRelevancy::ReasonGenerator
+    def self.call(...) = new(...).call
+
+    def initialize(question_message:, verdicts:, score:)
+      @question_message = question_message
+      @verdicts = verdicts
+      @score = score
+    end
+
+    def call
+      result = AutoEvaluation::BedrockOpenAIOssInvoke.call(user_prompt, json_schema)
```

Comment:
```suggestion
      result = BedrockOpenAIOssInvoke.call(user_prompt, json_schema)
```

Now we're within the AutoEvaluation module we can drop this extra namespacing (there's a bunch of subsequent examples)